src/cljs/rems/application_list.cljs

```diff
@@ -95,10 +97,29 @@
                            :rows [::table-rows applications]
                            :default-sort-column default-sort-column
                            :default-sort-order default-sort-order}]
-    [:div
-     (when filterable?
-       [table/search application-table])
-     [table/table application-table]]))
+    [table/table application-table]))
+
+(defn- application-list-defaults []
+  (let [config @(rf/subscribe [:rems.config/config])
+        id-column (get config :application-id-column :id)]
+    {:visible-columns #{id-column :description :resource :applicant :state :created :submitted :last-activity :view}
+     :default-sort-column :created
+     :default-sort-order :desc}))
+
+(defn default-component [{:keys [applications empty-message hidden-columns] :as opts}]
```

Comment:
docs please

also, I'm not sure I understand the name... how about renaming `component` to `list` and `default-component` to `component` or something?